app/src/renderer/components/common/ShortAddress.vue

```diff
@@ -0,0 +1,79 @@
+<template lang="pug">
+.short-address
+  .address(@click.prevent.stop="copy" v-tooltip.top="address")#address {{ shortAddress }}
+  .success(:class="{active:showSuccess}")
+    i.material-icons check
+    span Copied
+</template>
+
+<script>
+import { clipboard } from "electron"
+export default {
+  name: `short-address`,
+  props: [`address`],
+  data: () => ({
+    showSuccess: false
+  }),
+  computed: {
+    shortAddress({ address } = this, length = 4) {
+      if (!address) return `Address Not Found`
+      if (address.indexOf(`1`) === -1) {
```

Comment:
How can a bech32 address not have a 1 in it?

Reply:
not sure. i copied this code from `scripts/common`.